cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsCloudNioRegularFileAttributes.scala

```diff
@@ -1,48 +1,57 @@
 package cloud.nio.impl.drs
 
 import java.nio.file.attribute.FileTime
+import java.time.{LocalDateTime, ZoneOffset}
 
+import cats.effect.IO
 import cloud.nio.spi.CloudNioRegularFileAttributes
 import com.typesafe.config.Config
 
-import scala.concurrent.duration._
-
 
 class DrsCloudNioRegularFileAttributes(config: Config, drsPath: String, drsPathResolver: DrsPathResolver) extends CloudNioRegularFileAttributes{
 
   private def throwRuntimeException(missingKey: String) = {
-    throw new RuntimeException(s"Failed to resolve DRS path $drsPath. The response from Martha doesn't contain the key '$missingKey'.")
+    new RuntimeException(s"Failed to resolve DRS path $drsPath. The response from Martha doesn't contain the key '$missingKey'.")
+  }
+
+  private def timeInStringToFileTime(timeInString: String): FileTime = {
+    //Here timeInString is assumed to be a ISO-8601 DateTime without timezone
+    val instant = LocalDateTime.parse(timeInString).toInstant(ZoneOffset.UTC)
```

Comment:
pretty sure this parse can throw an exception?  (If so this function should be `IO`)

Reply:
Yes it does. I will wrap it around IO. Thanks!